script.js

```diff
@@ -0,0 +1,62 @@
+let bookList = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.